src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.iOS.cs

```diff
@@ -91,40 +91,6 @@ await CreateHandlerAndAddToWindow<CollectionViewHandler>(collectionView, async h
 			});
 		}
 
-		[Fact("Cells Do Not Leak")]
-		public async Task CellsDoNotLeak()
```

Comment:
I pushed a commit to restore the test.

Reply:
I intentionally left it out because it’s guaranteed to fail. See the post at the top of the PR for rationale.

Reply:
…but thanks for starting the CI, the big question is if the other tests fail, and it should give us the answer.

Reply:
So this used to be the circular reference (in https://github.com/dotnet/maui/commit/05697a6b1fc8a068ad537cbf455d2407e22e7e72 ~June 2023):
* `CollectionView` -> handlers / etc. -> `TemplatedCell` -> `LabelHandler` -> `Label` -> `CollectionView`

But now we have 10e3fc4db4866921b31cd04c72b05b3b2e76826a, so there is no strong reference from `Label` -> `CollectionView`.

If the old test is not valid, it seems like we should make sure some other test checks this.

Reply:
> If the old test is not valid, it seems like we should make sure some other test checks this.

Agreed. I wanted to have a full test run first to see if anything else fails. I'll have to think about bit on how to structure the test (or whether there is an existing test coverage).

Reply:
I updated the test to ensure that `CollectionView` can get collected. The `VerticalCell` is still referenced, so it means there's no (strong) cycle.